### ISSUE TITLE
FaceLandmarks is private

### ISSUE THREAD

**Issue by OhMyVolk** [podo-os/dlib-face-recognition#3]
`dlib_face_recognition::landmark_prediction::landmarks::FaceLandmarks` Is private and therefore the following code snippet won't compile.

```rust
use dlib_face_recognition::*;

fn get_landmarks(image_matrix: &ImageMatrix, rect: &Rectangle) -> FaceLandmarks {
    // doing stuff....
}
```

Also, thanks for all the work on the library!

**Comment by HoKim98** [comment]
Closed by merging #4.

**Comment by inferrna** [comment]
Please publish a new release with this fix. Bug currently present in version 0.2.2
